desktop/src/main/java/bisq/desktop/main/overlays/windows/DisputeSummaryWindow.java

```diff
@@ -963,7 +963,7 @@ private void applyUiControlsToDisputeResult(Toggle selectedTradeAmountToggle) {
 
         Coin penalizedPortionOfTradeAmount = Coin.ZERO;
         try {
-            disputeResult.setPayoutAdjustmentPercent(compensationOrPenalty.getText().replaceAll("[^0-9]", ""));
+            disputeResult.setPayoutAdjustmentPercent(compensationOrPenalty.getText().replaceAll("[^0-9,.]", ""));
```

Comment:
What about changing type of `payoutAdjustmentPercent` to `double` instead of `String`, and handling this `replaceAll` logic inside `ParsingUtils.parsePercentStringToDouble` method?

Reply:
@xyzmaker123 thankyou for the review.  However I sincerely believe in this case that it would be better remaining as is.